packages/nimble-components/src/rich-text-viewer/styles.ts

```diff
@@ -17,7 +18,7 @@ export const styles = css`
         inline-size: auto;
         overflow: auto;
         block-size: 100%;
-        min-block-size: 36px;
+        min-block-size: ${bodyFontLineHeight};
```

Comment:
Would you mind adding a default sizing test case with one line string to notice this new layout behavior in storybook?

Reply:
I have added a default height sizing test case for single line usage in storybook.
![image](https://github.com/ni/nimble/assets/123377523/2ed2a947-0998-4591-b581-d1983538103b)